ports/cubeb/portfile.cmake

```
@@ -0,0 +1,23 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO mozilla/cubeb
+    REF dc511c6b3597b6384d28949285b9289e009830ea
```

```suggestion
    REF d238150bbacccfb97e62499b0e79a834fcbbd102 # v0.2
```
There is a tag pointing to version 0.2.

---

Yes, but this tag is from 2012.

---

We should annotate the specific commit number date for easy updating and maintenance. 
I think this is better. :)